src/centauri/azext_centauri/commands.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+from azure.cli.core.commands import CliCommandType
+from azext_centauri._client_factory import cf_centauri
+
+
+def load_command_table(self, _):
+
+    # TODO: Add command type here
+    # centauri_sdk = CliCommandType(
+    #    operations_tmpl='<PATH>.operations#None.{}',
+    #    client_factory=cf_centauri)
+
+
+    with self.command_group('centauri') as g:

```

Comment:
Wouldn't we want the command group to be 'functionapp'?

Reply:
yes, we need to update that to `functionapp`. I will fix in next commit. Thanks